### ISSUE TITLE
Configurable application path

### ISSUE THREAD

**Issue by batusai513** [jaredpalmer/razzle#1459]
## 🚀 Feature request
Make the appPath configurable via ENV configuration

### Current Behavior

In order to run a razzle app razzle needs to be installed in the the package itself, and be run from there.

### Desired Behavior

I would like to make some sort of CLI package based on razzle on our internal monorepo, we are producing many several apps and on each of them we have to have installed razzle, so the idea is to leave this line configurable https://github.com/jaredpalmer/razzle/blob/master/packages/razzle/config/paths.js#L9, and default to `process.cwd()`, when setting this ENV variable we can send the folder path where the app is located.

### Suggested Solution

make https://github.com/jaredpalmer/razzle/blob/master/packages/razzle/config/paths.js#L9 configurable with an ENV variable and default to `process.cwd()`

```js
const appDirectory = fs.realpathSync(process.env.APP_PATH || process.cwd());
```

### Who does this impact? Who is this for?

This configuration will allow advanced users to create a CLI based on razzle, allowing them to have only one version of razzle for many multiple applications.

**Comment by fivethreeo** [comment]
good idea, added in canary now

**Comment by fivethreeo** [comment]
https://github.com/jaredpalmer/razzle/commit/53624cc48d52f00648b07de8ad579529820038e8

**Comment by fivethreeo** [comment]
Released in v4.0.0-canary.4
